nf_core/pipelines/lint/__init__.py

```diff
@@ -543,7 +544,7 @@ def run_linting(
     md_fn=None,
     json_fn=None,
     hide_progress: bool = False,
-) -> Tuple[PipelineLint, ComponentLint, Union[ComponentLint, None]]:
+) -> Tuple[PipelineLint, Union[ComponentLint, None], Union[ComponentLint, None]]:
```

Comment:
```suggestion
) -> Tuple[PipelineLint, Optional[ComponentLint], Optional[ComponentLint]]:
```
probably need to import that type in the top